weld/lib.rs

```diff
@@ -242,6 +242,15 @@ pub unsafe extern "C" fn weld_module_compile(code: *const c_char,
     let code = CStr::from_ptr(code);
     let code = code.to_str().unwrap().trim();
 
+    // Let LLVM find symbols in libweldrt; it's okay to call this multiple times
+    let libweldrt = if cfg!(target_os="macos") {
+        "libweldrt.dylib"
+    } else {
+        "libweldrt.so"
```

Comment:
Add a DLL for windows? Not sure if the rest of the code even works with Windows atm though